src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -1127,6 +1127,15 @@ public String execute(Jedis connection) {
       }
     }.run(null);
   }
+  @Override
+  public String readonly() {
```

Comment:
Readonly is fit for BasicCommand, and JedisCluster now implements BasicCommand.
Seems like we should merge cluster-revised PR to master first.